__tests__/base.js

```diff
@@ -95,6 +97,27 @@ var expectLevelSolved = function(levelBlob) {
   expectLevelAsync(headless, levelBlob);
 };
 
+var runCommand = function(command, resultHandler) {
+  var headless = new HeadlessGit();
+  var deferred = Q.defer();
+  var msg = null;
+
+  deferred.promise.then(function(commands) {
+    msg = commands[commands.length - 1].get('error').get('msg');
```

Comment:
i vaguely remember why i store all command output in the error field, but its certainly silly